File: trunk/jars/python/examples/stats.py
server = "www.twdata.org"
user = "mrdon"
prompt = ":~>"

ssh.connect(server, user);
ssh.setTimeout(20000);
ssh.waitFor(prompt);
ssh.sendLine("top");
print "==== Top ====\n"
print ssh.getLine()
print ssh.getLine()
print ssh.getLine()
print ssh.getLine()
ssh.send("^C")

ssh.waitFor(prompt)
print "\n==== Disk Usage ====\n"
ssh.sendLine("df")
mux = ["\r\n", prompt]
while 1:
    id = ssh.waitForMux(mux)
    if id == 0 :
        print ssh.lastLine()
    else:
        break
ssh.disconnect()

File: tags/0.4/jars/ruby/examples/stats.rb
server = "www.twdata.org"
user = "mrdon"
prompt = ":~>"

$ssh.connect(server, user);
$ssh.setTimeout(20000);
$ssh.waitFor(prompt);
$ssh.sendLine("top");
puts "==== Top ====\n"
puts $ssh.getLine()
puts $ssh.getLine()
puts $ssh.getLine()
puts $ssh.getLine()
$ssh.send("^C")

$ssh.waitFor(prompt)
puts "\n==== Disk Usage ====\n"
$ssh.sendLine("df")
mux = java.lang.reflect.Array.newInstance(java.lang.String, 2);
mux[0] = "\r\n"
mux[1] = prompt
while (true) 
	id = $ssh.waitForMux(mux)
	if id == 0 
		puts $ssh.lastLine()
	else
		retry
	end
end
$ssh.disconnect()
